node/cli/src/command.rs

```diff
@@ -62,21 +58,26 @@ impl SubstrateCli for Cli {
 	}
 
 	fn native_runtime_version(spec: &Box<dyn ChainSpec>) -> &'static RuntimeVersion {
-		if spec.is_pangoro() {
-			&pangoro_runtime::VERSION
-		} else {
+		#[cfg(feature = "template")]
+		if spec.is_template() {
+			return &template_runtime::VERSION;
+		}
+
+		if spec.is_pangolin() {
 			&pangolin_runtime::VERSION
+		} else {
+			&pangoro_runtime::VERSION
 		}
 	}
 
 	fn load_spec(&self, id: &str) -> Result<Box<dyn ChainSpec>, String> {
 		let id = if id == "" {
 			let n = get_exec_name().unwrap_or_default();
-			["pangolin", "pangoro"]
+			["template", "pangolin", "pangoro"]
 				.iter()
 				.cloned()
 				.find(|&chain| n.starts_with(chain))
-				.unwrap_or("pangolin")
+				.unwrap_or("pangoro")
```

Comment:
Why change `pangolin` to `pangoro` here?

Reply:
Default to Pangoro (Darwinia).